src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -63,6 +63,10 @@
 <p id="status">Keyboard Event Status</p>
 
 <button id="dealButton">Deal</button>
+<select id = "selectedGame" onchange = "display()">
+    <option value = "engGame">English Game</option>
```

Comment:
Can you please use the selected attribute so english is the default when the page loads.
Here is an example: http://www.w3schools.com/tags/att_option_selected.asp